tests/server/lib/applications.tests.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { expect } from 'chai';
+import { moveApplication } from '../../../server/lib/applications';
+
+let storageData = {
+  applications: {
+    app1: {
+      name: 'First App'
+    },
+    app2: {
+      name: 'First Two'
+    },
+    app3: {
+      name: 'First Three'
+    }
+  }
+};
+
+const storage = {
+  read: () => Promise.resolve(storageData),
+  write: (data) => {
+    storageData = data;
+    Promise.resolve();
+  }
+};
+
+describe('applications', () => {
```

Comment:
what if the `id` is not found in the list of applications?
what if the `direction` is not `up` or `down`?